### PR TITLE
Add stories for data extension editor

### PR DESCRIPTION
**Pull request by koesie10**
This adds stories for the two components included in the data extensions editor. This is dependent on #2257 and on #2258.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/.babelrc.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (charisk) [inline extensions/ql-vscode/.babelrc.json:1]: Since this is only used for Storybook, is it possible to have it in `/extensions/ql-vscode/.storybook`?

**Review by charisk — APPROVED** [review]
Great to see stories added for these components!

File: extensions/ql-vscode/.babelrc.json (hunk 1 above)
Comment (koesie10) [reply extensions/ql-vscode/.babelrc.json:1]: Unfortunately, that doesn't work and it will not compile.

Comment (charisk) [reply extensions/ql-vscode/.babelrc.json:1]: The issue is that now there is a `.babelrc` file sitting at the top which could confuse people as there are so many elements to compilation of the extension. And with the PR being squashed merged we lose the easy access to the history. I don't have a good suggestion right now, but it's something to think about how we could make better.

Comment (koesie10) [reply extensions/ql-vscode/.babelrc.json:1]: I agree that it's somewhat confusing, but I didn't squash merge this PR, so the original commit is now on `main` and shows why this was added: https://github.com/github/vscode-codeql/commit/bb05220b2c5acd906d9d61ba963aff17c3d9373b

Comment (charisk) [reply extensions/ql-vscode/.babelrc.json:1]: Oh nice sorry I misread the history!

**Pull request merged**